hydra/test_utils/test_utils.py

```diff
@@ -277,8 +277,16 @@ def integration_test(
     filename: str = "task.py",
     env_override: Dict[str, str] = {},
     clean_environment: bool = False,
+    generate_custom_cmd: Callable[..., List[str]] = lambda *args, **kwargs: [],
 ) -> str:
-    Path(tmpdir).mkdir(parents=True, exist_ok=True)
+
+    modified_tmpdir = tmpdir
+    if "tmpdir" in task_config:
+        modified_tmpdir = Path(task_config["tmpdir"] + str(tmpdir))
```

Comment:
now that you can customize the scratch dir on the caller, you should no longer need this.

Reply:
fixed all occurrences

Reply:
in the new commit